src/main/java/com/ldtteam/domumornamentum/entity/block/MateriallyTexturedBlockEntity.java

```diff
@@ -41,19 +39,7 @@ public void updateTextureDataWith(final MaterialTextureData materialTextureData)
     @Override
     public @NotNull CompoundTag getUpdateTag()
     {
-        return this.saveWithId();
-    }
-
-    @Override
-    public void onDataPacket(final Connection net, final ClientboundBlockEntityDataPacket packet)
```

Comment:
This is the client server sync, why remove it?

Reply:
override without anychange

Reply:
ah okay, this is the vanilla default. I always thought this was sort of empty by default =P

Reply:
both are forge additions with default impl